src/main/KeyboardControls.tsx

```diff
@@ -95,20 +88,18 @@ const Entry: React.FC<{params: KeyMapDisplayOptions}> = ({params}) => {
 
   return (
     <div css={entryStyle}>
-      <div css={labelStyle}><Trans>{params.name || t("keyboardControls.missingLabel")}</Trans></div>
-      <div css={sequencesStyle}>
-        {params.sequences.map((sequence, index, arr) => (
-          <div css={sequenceStyle} key={index}>
-            {sequence.sequence.toString().split('+').map((singleKey, index, {length}) => (
-              <>
-                <div css={singleKeyStyle} key={index}>{singleKey}</div>
-                {length - 1 !== index ? <div css={orStyle}>+</div> : ''}
-              </>
-            ))}
-            <div css={orStyle}><Trans>{arr.length - 1 !== index && t("keyboardControls.sequenceSeparator")}</Trans></div>
-          </div>
-        ))}
-      </div>
+      <div css={labelStyle}><Trans>{name || t("keyboardControls.missingLabel")}</Trans></div>
+      {sequences.map((sequence, index, arr) => (
+        <div css={sequenceStyle} key={index}>
```

Comment:
You swapped the order of some stuff so that one `<div css={sequencesStyle}>` is created per sequence. This means each seqeuence is in its own line, which is different from before. It becomes clear what I mean when you compare the shortcut overview pages of the current main branch and this.

Reply:
That is intended. I want to render each sequence in its own line to help with readability. This was discussed in some other issue, and I simply took the opportunity since I was working on that code anyway. But admittedly, this has nothing to do with updating to React 18, so I can try to revert those changes and make a separate PR for them, should you deem it desirable.

Reply:
Ah I understand. Yeah no problem, keep it as part of this PR, I don't mind. Maybe the PR description should mention that though and maybe link to said discussion.